app/sources/HFTemplateController.m

```diff
@@ -399,6 +399,26 @@ - (NSString *)readFatDateWithLabel:(NSString *)label error:(NSString **)error {
     return date;
 }
 
+- (NSString *)readFatTimeWithLabel:(NSString *)label error:(NSString **)error {
+    int16_t val;
+    if (![self readInt16:&val forLabel:nil]) {
+        if (error) {
+            *error = @"Failed to read int16 bytes";
+        }
+        return nil;
+    }
+
+    int sec = (val & 0x1F) * 2;
+    int min = (val >> 5) & 0x3F;
+    int hour = (val >> 11) & 0x1F;
+    NSString *time = [NSString stringWithFormat:@"%02d:%02d:%02d UTC", hour, min, sec];
```

Comment:
Is it UTC? Even so, I'm not sure it makes sense to include that with the time format.

Reply:
I am merging, but will remove UTC.